tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -1122,16 +1059,10 @@ String updateIntLocs(int id, boolean doCommit, Connection conn, PreparedStatemen
         removeIntLoc.setInt(1, id);
         removeIntLoc.executeUpdate();
 
-        String[] il = user.getInterestList();
-        if (il != null) {
-            updateIntLoc.setInt(1, id);
```

Comment:
this line was inadvertently dropped

Reply:
You're right! I see you've fixed it in https://github.com/unicode-org/cldr/pull/1348